app/providers/orders/order.ts

```diff
@@ -85,7 +85,7 @@ gql`
   }
 `;
 gql`
-  mutation setOrderShippingMethod($shippingMethodId: ID!) {
+  mutation setOrderShippingMethod($shippingMethodId: [ID!]!) {
```

Comment:
Careful here, I only "fixed" the type so we can compile but havent looked into the usage yet.

Reply:
Mutation appearently accepts `string | string[]` so we're good here